File: react-app/src/components/testData/defaultList.ts
import { DefaultListType, DB_DefaultListType } from '../type/DefaultList';
import { stockList } from './stockList';

export const defaultList: DefaultListType[] = [
    {
        id: 1,
        stockList: stockList[0],
        amount: 1
    },
    {
        id: 2,
        stockList: stockList[1],
        amount: 2
    },
    {
        id: 3,
        stockList: stockList[2],
        amount: 3
    },
    {
        id: 4,
        stockList: stockList[3],
        amount: 4
    }
];
export const DB_defaultList: DB_DefaultListType[] = [
    {
        id: 1,
        stockList_id: 1,
        amount: 1
    },
    {
        id: 2,
        stockList_id: 2,
        amount: 2
    }
];

File: python/api/models/default_list.py
from sqlalchemy import Column, Integer, ForeignKey, DateTime, Boolean
from sqlalchemy.sql import func

from db import Base

class DefaultList(Base):
    __tablename__ = "default_list"
    id = Column(Integer, primary_key=True,autoincrement=True)
    stock_list_id = Column(Integer,ForeignKey("stock_list.id") ,nullable=False)
    amount = Column(Integer, nullable=False, default=0)
    isAccept = Column(Boolean, nullable=False, default=False)
    created_at = Column(DateTime, server_default=func.now())